tests/test_redirect_landing.py

```diff
@@ -11,17 +11,25 @@
 @pytest.mark.skip_selenium
 class TestRedirectLanding(object):
     LOCALES = (
-        "ach", "af", "ak", "ar", "as", "ast", "be", "bg",
-        "bn-BD", "bn-IN", "br", "bs", "ca", "cs", "csb", "cy", "da",
-        "de", "el", "en-GB", "en-ZA", "eo", "es-AR", "es-CL", "es-ES",
-        "es-MX", "et", "eu", "fa", "ff", "fi", "fr", "fy-NL", "ga-IE",
-        "gd", "gl", "gu-IN", "he", "hi-IN", "hr", "hu", "hy-AM", "id",
-        "is", "it", "ja", "kk", "km", "kn", "ko", "ku", "lg", "lij", " lt",
-        "lv", "mai", "mk", "ml", "mr", "nb-NO", "nl", "nn-NO", "nso",
-        "or", "pa-IN", "pl", "pt-BR", "pt-PT", "rm", "ro", "ru", "si", "sk",
-        "sl", "son", "sq", "sr", "sv-SE", "sw", "ta", "ta-LK", "te", "th", "tr",
-        "uk", "vi", "zh-CN", "zh-TW", "zu"
+        'af', 'an', 'ar', 'as', 'ast', 'az', 'be', 'bg', 'bn-IN', 'br', 'ca',
+        'cs', 'cy', 'da', 'de', 'dsb', 'el', 'en-GB', 'en-US', 'eo', 'es-AR',
+        'es-CL', 'es-ES', 'es-MX', 'et', 'eu', 'fa', 'ff', 'fi', 'fr', 'fy-NL',
+        'ga-IE', 'gd', 'gl', 'he', 'hi-IN', 'hr', 'hsb', 'hu', 'hy-AM', 'id',
+        'is', 'it', 'ja', 'ka', 'kk', 'km', 'ko', 'lij', 'lt', 'lv', 'mk', 'ml',
+        'mr', 'ms', 'my', 'nb-NO', 'nl', 'nn-NO', 'oc', 'pa-IN', 'pl', 'pt-BR',
+        'pt-PT', 'rm', 'ro', 'ru', 'sk', 'sl', 'son', 'sq', 'sr', 'sv-SE', 'ta',
+        'te', 'th', 'tr', 'uk', 'ur', 'uz', 'vi', 'xh', 'zh-CN', 'zh-TW',
     )
+    LOCALE_VARIANTS = {
```

Comment:
Small nit, can you add a code comment defining what `LOCALE_VARIANTS` is for and how it differs from `LOCALES`?